EpiAware/src/models.jl

```diff
@@ -42,16 +11,16 @@ A named tuple containing the generated quantities `I_t` and `latent_process_para
     neg_bin_cluster_factor ~ neg_bin_cluster_factor_prior
 
     #Latent process
-    time_steps = epimodel.time_horizon
-    @submodel _I_t, latent_process_parameters =
+    time_steps = epimodel.data.time_horizon
+    @submodel latent_process, latent_process_parameters =
         latent_process(time_steps; latent_process_priors = latent_process_priors)
 
     #Transform into infections
-    I_t = transform_function.(_I_t)
+    I_t, _ = scan(epimodel, latent_process_parameters.init, latent_process)
 
     #Predictive distribution
     case_pred_dists =
-        (epimodel.delay_kernel * I_t) .+ pos_shift .|>
+        (epimodel.data.delay_kernel * I_t) .+ pos_shift .|>
```

Comment:
As currently coded isn't the neg_bin_cluster_factor part of EpiData?

Reply:
That should probably be `true_neg_bin_cluster_factor`; I was thinking about storing all the data required to generate data from a "true model".

Reply:
I'm still a bit confused about this as I don't think it should be part of the `EpiData` as its an observation process but  I think we should discuss in another issue.